### PR TITLE
Create distinct openj9-licenses and openj9-notices

### PR DESCRIPTION
**Pull request by pshipton**
Issue https://github.com/eclipse-openj9/openj9/issues/17164

Depends on https://github.com/eclipse-openj9/openj9/pull/17166

Will port to other versions after the initial review.

### CODE REVIEW

File: closed/custom/modules/java.base/Copy.gmk [hunk 1]
```diff
@@ -167,7 +167,10 @@ $(call openj9_copy_files,, \
 
 $(call openj9_copy_files,, \
 	$(OPENJ9_TOPDIR)/LICENSE \
-	$(LEGAL_DST_DIR)/openj9-notices)
+	$(LEGAL_DST_DIR)/openj9-licenses)
+$(call openj9_copy_files,, \
+	$(OPENJ9_TOPDIR)/NOTICES.md \
+	$(LEGAL_DST_DIR)/openj9-notices.md)
```
Comment (keithc-ca) [inline closed/custom/modules/java.base/Copy.gmk:173]: While you're here, could you please fix the indentation of lines 178,179 to use tabs like other uses of `openj9_copy_files`?

Comment (pshipton) [reply closed/custom/modules/java.base/Copy.gmk:173]: Updated

**Comment by pshipton** [comment]
Backports created
https://github.com/ibmruntimes/openj9-openjdk-jdk20/pull/38
https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/201
https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/643
https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/659

**Comment by keithc-ca** [comment]
Jenkins compile amac jdknext

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**